tools/maven/suppressions.xml

```diff
@@ -32,13 +32,6 @@ under the License.
 		<suppress files="org[\\/]apache[\\/]beam[\\/]vendor[\\/]grpc[\\/]v1p21p0[\\/]io[\\/]netty[\\/]buffer.*.java" checks="[a-zA-Z0-9]*"/>
 		<suppress files="org[\\/]apache[\\/]beam[\\/]runners[\\/]fnexecution[\\/]environment.*.java" checks="[a-zA-Z0-9]*"/>
 
-		<!-- Python streaming API follows python naming conventions -->
-		<suppress
-			files="org[\\/]apache[\\/]flink[\\/]streaming[\\/]python[\\/]api[\\/].*.java"
-			checks="MethodNameCheck|ParameterName"/>
-		<suppress
-			files="org[\\/]apache[\\/]flink[\\/]streaming[\\/]python[\\/]connectors[\\/].*.java"
-		checks="MethodNameCheck|ParameterName"/>
```

Comment:
Why can we remove these suppressions? According to the comment, the Java Python files follow python naming conventions?

Reply:
Is it because these files are only in the `target` directory?

Reply:
those are part of the python streaming api we removed a long time ago.

Reply:
Thanks for the clarification.